### PR TITLE
[Chat][File Sharing] Renamed Files to Attachments

### PR DESCRIPTION
**Pull request by jpeng-ms**
# What

This PR renames file components to attachment components without public API changes.

Additional PR would be created to:
- rename files in ChatMessage
- rename Composite options (fileSharing)
- rename data models (fileMetadata)

# Why
general refactor

# How Tested
CI

# Process & policy checklist

- [X] I have updated the project documentation to reflect my changes if necessary.
- [X] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/AttachmentUploadCards.tsx [hunk 1]
```diff
@@ -48,11 +48,11 @@ export interface ActiveFileUpload {
...
 /**
- * Strings of _FileUploadCards that can be overridden.
+ * Strings of _AttachmentUploadCards that can be overridden.
  *
  * @internal
  */
-export interface _FileUploadCardsStrings {
+export interface _AttachmentUploadCardsStrings {
   /** Aria label to notify user when focus is on cancel file upload button. */
   removeFile: string;
```
Comment (emlynmac) [inline packages/react-components/src/components/AttachmentUploadCards.tsx:57]:
```suggestion
  removeAttachment: string;
```

File: packages/react-components/src/components/AttachmentUploadCards.tsx [hunk 2]
```diff
@@ -78,15 +78,15 @@ export interface FileUploadCardsProps {
...
 }
 
 const actionIconStyle = { height: '1rem' };
 
 /**
  * @internal
  */
-export const _FileUploadCards = (props: FileUploadCardsProps): JSX.Element => {
+export const _AttachmentUploadCards = (props: FileUploadCardsProps): JSX.Element => {
   const files = props.activeFileUploads;
```
Comment (emlynmac) [inline packages/react-components/src/components/AttachmentUploadCards.tsx:90]:
```suggestion
  const attachments = props.activeFileUploads;
```

File: packages/react-components/src/components/AttachmentUploadCards.tsx [hunk 3]
```diff
@@ -103,12 +103,12 @@ export const _FileUploadCards = (props: FileUploadCardsProps): JSX.Element => {
   }
 
   return (
-    <_FileCardGroup>
+    <_AttachmentCardGroup>
       {files &&
         files
           .filter((file) => !file.error)
```
Comment (emlynmac) [inline packages/react-components/src/components/AttachmentUploadCards.tsx:109]: file=>attachment

**Review by jimchou-dev — APPROVED** [review]

File: packages/react-components/src/components/AttachmentUploadCards.tsx (hunk 1 above)
Comment (jpeng-ms) [reply packages/react-components/src/components/AttachmentUploadCards.tsx:57]: i am not sure, apparently this name has been GA-ed? 

image

**Review by emlynmac — APPROVED** [review]

**Pull request merged**